src/base/jstemplates/place-detail-edit-bar.html

```diff
@@ -0,0 +1,25 @@
+<!-- Template for the row of editor buttons visible at the top of place
+    detail views. -->
+
+{{#if isEditable}}
```

Comment:
I debated this as well. My reasoning for having the `{{#if}}` check inside the template here is that the check is integral to the way this template should behave. In other words, there's no situation where we wouldn't want to render this template without the `{{#if}}` check.